packages/main/src/themes/Badge.css

```diff
@@ -40,9 +39,9 @@
 	font-weight: inherit;
 	text-overflow: ellipsis;
 	line-height: 1;
-	text-transform: var(--_ui5-badge-text-transform);
+	text-transform: uppercase;
 	letter-spacing: inherit;
-	font-size: var(--ui5-badge-font-size); /* origin from --sapFontSmallSize (0.75rem) */
+	font-size: 0.75em; /* origin from --sapFontSmallSize (0.75rem) */
```

Comment:
Why don't we use the variable var(--sapFontSmallSize?

Reply:
Because it in `rem`. Badge uses `em` since https://github.com/SAP/ui5-webcomponents/pull/729, I didn't want to change that